raiden/api/python.py

```diff
@@ -65,6 +65,32 @@ def get_channel(self, channel_address):
 
         raise ValueError("Channel not found")
 
+    def manager_address_if_token_registered(self, token_address):
+        """
+        If the token is registered then, return the channel manager address.
+        Returns None otherwise.
+        """
+        try:
+            manager = self.raiden.chain.manager_by_token(token_address)
```

Comment:
Found where I was messing: I was confusing `BlockChainService.manager_by_token` with `Registry.manager_address_by_token`. The later which returns `''` when token isn't registered, which then causes the former to fail on the next call, trying to open the channel with empty address.
Ideally, we could raise some exception directly on `manager_address_by_token`, to save the try to open the channel and following `JSONRPCClientReplyError`, but it is ok to get it after it tries to open the channel too. Thank you.